server/foreignPublication/model.js

```diff
@@ -0,0 +1,11 @@
+export default (sequelize, dataTypes) => {
+	return sequelize.define('ForeignPublication', {
+		title: { type: dataTypes.TEXT, allowNull: false },
+		url: { type: dataTypes.TEXT, allowNull: false },
+		byline: { type: dataTypes.TEXT },
+		doi: { type: dataTypes.TEXT },
+		description: { type: dataTypes.TEXT },
+		avatar: { type: dataTypes.TEXT },
+		publicationDate: { type: dataTypes.DATE },
+	});
```

Comment:
One thought that could be out of scope for this PR is to consider what [open graph tags](https://ogp.me/) we expect most publications to support that would be useful. Would we want to grab `of:type` to note when something is a video, for example? Perhaps `image:alt`?

Reply:
Maybe! But definitely out of scope here.